src/main/java/org/broad/igv/sam/AlignmentTrack.java

```diff
@@ -367,30 +368,22 @@ public void init() {
 
 
     @Override
-    public void receiveEvent(Object event) {
+    public void receiveEvent(IGVEvent event) {
 
         if (event instanceof FrameManager.ChangeEvent) {
             // Trim insertionInterval map to current frames
 
-
-        } else if (event instanceof AlignmentTrackEvent) {
-            AlignmentTrackEvent e = (AlignmentTrackEvent) event;
-            AlignmentTrackEvent.Type eventType = e.getType();
-            switch (eventType) {
-                case ALLELE_THRESHOLD:
-                    dataManager.alleleThresholdChanged();
-                    break;
-                case RELOAD:
+        } else if (event instanceof AlignmentTrackEvent e) {
+            switch (e.type()) {
+                case ALLELE_THRESHOLD -> dataManager.alleleThresholdChanged();
+                case RELOAD -> {
                     clearCaches();
                     repaint();
-                case REFRESH:
-                    repaint();
-                    break;
+                }
+                case REFRESH -> repaint();
```

Comment:
There was a silly bug here where on a RELOAD we would repaint twice since it would fall through without a `break`